plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -237,6 +359,14 @@ class HealthChecksFormSection extends Component {
               onClick={this.props.onAddItem.bind(this, {value: data.healthChecks.length, path: 'healthChecks'})}>
               <Icon color="purple" id="plus" size="tiny" /> Add Health Check
             </a>
+            <Tooltip
```

Comment:
I believe this tooltip should be adjacent to the section heading: "Health Checks" (at least that's is how I read the docs).